CMakeLists.txt

```diff
@@ -14,22 +14,23 @@ endif()
 
 # Imath version
 
-project(Imath VERSION 3.0.2 LANGUAGES C CXX)
+project(Imath VERSION 3.1.0 LANGUAGES C CXX)
 
 set(IMATH_VERSION_EXTRA "dev" CACHE STRING "Extra version tag string for Imath build")
 
 # See https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html
-set(IMATH_SOVERSION 28)
+set(IMATH_SOVERSION 29)
 set(IMATH_SOREVISION 0) 
 set(IMATH_SOAGE 0) 
 set(IMATH_LIB_VERSION "${IMATH_SOVERSION}.${IMATH_SOREVISION}.${IMATH_SOAGE}")
 
-set(IMATH_VERSION_MAJOR ${CMAKE_PROJECT_VERSION_MAJOR})
-set(IMATH_VERSION_MINOR ${CMAKE_PROJECT_VERSION_MINOR})
-set(IMATH_VERSION_PATCH ${CMAKE_PROJECT_VERSION_PATCH})
-set(IMATH_VERSION ${CMAKE_PROJECT_VERSION})
+set(IMATH_VERSION_MAJOR ${Imath_VERSION_MAJOR})
+set(IMATH_VERSION_MINOR ${Imath_VERSION_MINOR})
+set(IMATH_VERSION_PATCH ${Imath_VERSION_PATCH})
```

Comment:
Not for this patch -- because I'm hesitant to introduce any potentially breaking change to the backports -- but if IMATH_VERSION_MAJOR is never different from Imath_VERSION_MAJOR, do we really need the former to exist at all?

Reply:
Agreed, not for this patch. I created a v3.0.3 tag which will suffice for the current issue. I'll dig a little deeper afterwards.

Reply:
I removed the IMATH_VERSION_{MAJOR,MINOR,PATCH} variables and replaced their use with the Imath_ ones.

It bugs me that we have some variables prefixed with IMATH_ and some (cmake-provided) prefixed with Imath_, but I'm leery of changing the ones that are already public settings.

Reply:
Oh geez, I had to look up if cmake variables are case sensitive or not (they are case sensitive). This pattern is moderately confusing.